deploy/docker-compose/.env

```diff
@@ -0,0 +1,76 @@
+# Container Images
+INFORMATICS_GATEWAY_IMAGE=ghcr.io/project-monai/monai-deploy-informatics-gateway:0.3.0-rc.1
+INFORMATICS_GATEWAY_CONFIG_IMAGE=dwdraju/alpine-curl-jq:latest
+WORKFLOW_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-workflow-manager:develop-nightly-2022-09-12
+TASK_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-task-manager:develop-nightly-2022-09-12
+
+MINIO_IMAGE=minio/minio:latest
+MINIO_MC_IMAGE=minio/mc
+MONGODB_IMAGE=mongo:latest
+ORTHANC_IMAGE=osimis/orthanc
+RABBITMQ_IMAGE=rabbitmq:3-management
+
+
+# Default Listening AE Title
+INFORMATICS_GATEWAY_AE_TITLE=MONAI-DEPLOY
+# Service ports
+# Informatics Gatewway Ports - Changing the following values requires update to informatics-gateway.json
+INFORMATICS_GATEWAY_SCP_PORT=104
+INFORMATICS_GATEWAY_HL7_PORT=2575
+INFORMATICS_GATEWAY_API_PORT=5000
+# End Informatics Gatewway Ports
+
+WORKFLOW_MANAGER_PORT=5001
+TASK_MANAGER_PORT=5002
+
+MINIO_API_PORT=9000
+MINIO_CONSOLE_PORT=9001
+MONGODB_PORT=27017
+
+# Orthanc Ports- Changing the following values requires update to orthanc.json
+ORTHANC_SCP_PORT=4242
+ORTHANC_UI_PORT=8042
+# End Orthanc Ports
+
+RABBITMQ_MGT_PORT=15672
+RABBITMQ_PORT=5672
+
+# Static IP allocations & hostnames
+# Changing of IPs/host names require updates to the service configuration files
+SUBNET=172.29.0.0/16
+GATEWAY=172.29.0.1
```

Comment:
Is there a way to generate this IP address instead of manually editing it?

Reply:
the IPs are for communication between the containers, user shouldn't need to modify them unless they have conflicts.